hw_diag/diagnostics/env_var_diagnostics.py

```diff
@@ -19,6 +19,9 @@
 }, {
     'ENV_VAR': 'VARIANT',
     'DIAGNOSTIC_KEY': 'VA'
+}, {
```

Comment:
I think part of the confusion was that this array structure is pretty confusing (I wrote it originally, whoops). You have to dig through the code to understand that the `ENV_VAR` is used as the friendly_key and that `DIAGNOSTIC_KEY` is the key. 

Not worth creating a ticket just to refactor, but the next time one of us touches this code, it might be worth refactoring to something like:

```
[
...
  {
    'ENV_VAR': 'FREQ',
    'DIAGNOSTIC_KEY': 'FR'
    'DIAGNOSTIC_FRIENDLY_KEY': 'frequency'
  }
]
```

What do you think?